rules/windows/persistence_via_update_orchestrator_service_hijack.toml

```
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/08/17"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/17"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects potential hijack of Microsoft Update Orchestrator Service to persist a rogue program with system Integrity
+Level.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Persistence via Update Orchestrator Service Hijack"
+references = ["https://github.com/irsl/CVE-2020-1313"]
+risk_score = 73
+rule_id = "265db8f5-fc73-4d0d-b434-6483b56372e2"
+severity = "high"
+tags = ["Elastic", "Windows"]
+type = "query"
+
+query = '''
+event.category:process and not event.type:end and
```

```suggestion
event.category:process and event.type:(start or process_started) and
```

for process events, I would be explicit be using "start" because there is also an event.type of info which may result in unrelated or duplicative alerts. process_started is for backwards compat for pre 7.8 beats